dandiapi/api/models/asset_paths.py

```diff
@@ -24,8 +24,11 @@ class AssetPath(models.Model):
 
     class Meta:
         constraints = [
-            # Disallow slashses in path
-            models.CheckConstraint(check=~models.Q(path__endswith='/'), name='consistent-slash'),
+            # Disallow slashses at the beginning or end of path
```

Comment:
```suggestion
            # Disallow slashes at the beginning or end of path
```
😄 I'm surprised our codespell check didn't catch this

Reply:
(all that to say: `slashses` probably is not a "common misspelling" 🤷🏾)

Reply:
Ah, that makes a lot more sense